src/ipvs/ip_vs_conn.c

```diff
@@ -1150,6 +1166,12 @@ void dp_vs_conn_put_no_reset(struct dp_vs_conn *conn)
 /* put back the conn and reset it's timer */
 void dp_vs_conn_put(struct dp_vs_conn *conn)
 {
+    if ((conn->flags & DPVS_CONN_F_ONE_PACKET) &&
+            (rte_atomic32_read(&conn->refcnt) == 1) &&
+            !dp_vs_conn_is_in_timer(conn)) {
```

Comment:
I think `(rte_atomic32_read(&conn->refcnt) == 1)` and `!dp_vs_conn_is_in_timer(conn)` are not needed here.

Reply:
1. The conditions ` (rte_atomic32_read(&conn->refcnt) == 1) &&  !dp_vs_conn_is_in_timer(conn)` should always be true. You can verify it with `assert`. If no problem after test, it's advised to remove the unecessary conditions here because `dp_vs_conn_put` impacts performance a lot.
2. `dp_vs_conn_put_nolock` should modified accordingly.